<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="65832" started-at="2015-05-04T02:51:46Z" finished-at="2015-05-04T02:52:52Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="65832" started-at="2015-05-04T02:51:46Z" finished-at="2015-05-04T02:52:52Z">
      <class name="test.TestCase_man_NewCustomer">
        <test-method status="FAIL" signature="TestNewCustomer()[pri:0, instance:test.TestCase_man_NewCustomer@42bbf4]" name="TestNewCustomer" duration-ms="65808" started-at="2015-05-04T10:51:46Z" finished-at="2015-05-04T10:52:52Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[can not type in search bar]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: can not type in search bar
	at test.TestCase_man_NewCustomer.TestNewCustomer(TestCase_man_NewCustomer.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestNewCustomer -->
      </class> <!-- test.TestCase_man_NewCustomer -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
